src/components/MainPage.js

```diff
@@ -59,48 +56,70 @@ type Props = {
 
 type State = {
   inputValue: number | string,
-  favorites: Array<number>,
+  activePage: number,
+  favorites: string,
 };
 
 class MainPage extends Component<Props, State> {
   componentDidMount() {
-    this.props.getMovies.movies();
+    this.props.getMovies.movies(null, 1);
+    this.setState({ activePage: 1 });
+    localStorage.setItem('favorites', 'init');

```

Comment:
I think pagination must be handled thru url, some kind of `?page=1`, etc.

Reply:
Done